man/systemd-stub.xml

```diff
@@ -142,11 +144,25 @@
       details on system extension images. The generated <command>cpio</command> archive containing these
       system extension images is measured into TPM PCR 13 (if a TPM is present).</para></listitem>
 
+      <listitem><para>Similarly, files
+      <filename><replaceable>foo</replaceable>.efi.extra.d/*.addon.efi</filename>
+      are loaded and verified as PE binaries, and a <literal>.cmdline</literal> section is parsed from them.
+      In case Secure Boot is enabled, these files will be validated using keys in DB or MoK, and will be
```

Comment:
it's usually spelled SecureBoot (without space)

Measured Boot otoh is usually spelled with a space. dunno why, but we mostly followed this so far afaics.

Reply:
maybe DB → "UEFI DB" and MOK → "shim MOK"

Reply:
I could swear it was abbreviated as MoK - fixed